src/selectedtests/work_items/work_items_cli.py

```diff
@@ -82,16 +89,16 @@ def create_test_mapping(ctx: Context, project: str, src_regex: str, test_file_re
 
 @cli.command()
 @click.option(
-    "--weeks-back", type=int, default=DEFAULT_WEEKS_BACK, help="Number of weeks back to process."
+    "--years-back", type=int, default=DEFAULT_YEARS_BACK, help="Number of years back to process."
```

Comment:
[suggestion] Values like weeks back and years back float around a bit and change over time (so when the years back was can be difficult to figure out later).

Why not just provide a start data and then leverage linux command line tools, e.g.:

```
$ date  --date="3 years ago" -I
2017-03-06

$ date  --date="24 weeks ago" -I
2019-09-20

$ work-items process-task-mappings  --start-date "$(date  --date="24 weeks ago" -I)"
$ work-items process-task-mappings  --start-date "$(date  --date="3 years ago" -I)"
```
 
This makes the CLI more flexible (and hope fully clear).

Reply:
Interesting, I didn't know about the `date` linux command, that's cool.

There are two concerns I have with making this change:
* Right now we are always running `work-items process-task-mappings` on a constant date range (and this provides consistency across the different projects we seed mappings for), so I don't want to make the CLI more flexible until it's needed.
* Occasionally I like to be able to run these scripts from my Mac when debugging. Adding Linux commands to the  `cronjobs/process_task_mapping_work_items.sh` script would make it only able to run on Linux. (We could add some logic to work on both Mac and Linux, but I don't think this is worth it right now.)

Reply:
The date range would still be constant. 
The date cli should also work on a Mac.

It's only a suggestion so I have no problem ignoring it, we could raise a ticket to discuss it.